src/main/java/uk/gov/digital/ho/pttg/api/ArchiveResource.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.digital.ho.pttg.api;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.digital.ho.pttg.ArchiveService;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static uk.gov.digital.ho.pttg.application.LogEvent.*;
+
+@RestController
+@Slf4j
+public class ArchiveResource {
+
+    private ArchiveService archiveService;
+    private RequestData requestData;
+
+    public ArchiveResource(ArchiveService archiveService, RequestData requestData) {
+        this.archiveService = archiveService;
+        this.requestData = requestData;
+    }
+
+    @PostMapping("/nino/{nino}/archive/{result}")
```

Comment:
As discussed we should treat the archive as the resource i.e. the route should be /archive/{date} and the request body should contain the resutl and the eventIDs. Nino is not required.